Formula/s/sui.rb

```diff
@@ -0,0 +1,28 @@
+class Sui < Formula
+  desc "Next-generation smart contract platform powered by the Move programming language"
+  homepage "https://sui.io"
+  url "https://github.com/MystenLabs/sui/archive/refs/tags/testnet-v1.17.3.tar.gz"
+  sha256 "815e6acf5756e7eeab7fb1494642e3d26a69a66da260deac8efac82c5b280771"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^testnet[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "libpq" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "llvm" => :build
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "crates/sui")
+  end
+
+  test do
+    system "#{bin}/sui", "keytool", "list"
```

Comment:
can we also assert the response?

Reply:
There's no response if there's no available keystore for the tool, so I've adjusted the test to use a placeholder keystore